packages/agents/router/src/config.ts

```diff
@@ -15,6 +15,7 @@ const DEFAULT_SUBGRAPH_POLL_INTERVAL = 15_000;
 const DEFAULT_CONFIRMATIONS = 3;
 const MIN_CACHE_POLL_INTERVAL = 2_000;
 const DEFAULT_CACHE_POLL_INTERVAL = 20_000;
+const DEFAULT_MAX_PATH_LENGTH = 4;
```

Comment:
Change this to 3 for now

current on-chain maxRoutersPerTransfer is 5

Round 4 is 8-router transfer, and will revert

We can update this default to be 4 later